goalc/compiler/compilation/Type.cpp

```diff
@@ -89,6 +90,33 @@ RegVal* Compiler::compile_get_method_of_object(const goos::Object& form,
     }
   }
 
+  if (m_settings.check_for_requires) {
+    std::vector<symbol_info::SymbolInfo*> symbol_info =
+        m_symbol_info.lookup_exact_method_name(method_info.name, compile_time_type.base_type());
+    // TODO - check for virtual state requirement, if i could track the defining form!
+    // if (symbol_info.empty()) {
+    //  // maybe it's a virtual state
```

Comment:
yeah - I think this would be easier if we tracked this in the method info's. But don't worry about it for now